RELEASE_NOTES.md

```diff
@@ -62,6 +62,10 @@ Upgrade notes:
 	  as necessary. Single-ended and symmetric paired-end parameters will be
 	  upgraded automatically. These are often used for 10X runs.
 
+BAD:
+
+  * Confirm password field shows an error and prevents changing passwords and creating new users
```

Comment:
This would mean we should go back and do the same for every bug. I think it's easy enough to scroll up and find the first version that doesn't have the "BAD" in it

Reply:
Sure